tests/service/connectors/test_queryconfig.py

```diff
@@ -189,9 +189,9 @@ def test_update_rule_target_fields(
 
         # Make target more broad
         target = rule.targets[0]
-        target.data_category = DataCategory("user.provided.identifiable").value
+        target.data_category = DataCategory("user.contact").value
```

Comment:
Is this what you intended to do? Swapping user.provided.identifiable for user.contact?

Reply:
don't believe I did this on purpose (my memory isn't trustworthy), probably a sloppy find/replace

Reply:
I remember now that I did it to try and understand what the test was doing/what would pass. have reverted to the `user`